debugtools/DDR_VM/generate.xml

```diff
@@ -542,7 +542,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 				<fileset dir="${BLOBS_DIR}/../supersets" includes="*" />
 			</path>
 			<sequential>
-				<propertyregex property="specsuperset" override="true" input="@{file}" regexp="\/bluebird\/builds\/bld_[0-9]*\/build\/supersets/([^\.]*)" select="\1" casesensitive="false" />
+				<propertyregex property="specsuperset" override="true" input="@{file}" regexp="\/bluebird\/builds\/bld_[0-9]*\/build\/supersets/([^\.]*)" select="\1" casesnsitive="false" />
```

Comment:
This is not a good change. There are others similar.

Reply:
Dropped in https://github.com/jsoref/openj9/commit/cb93981cbfc17b6a8c16522178217ba6cbd3678d

This is the hazard of the way my tooling works. They're search and replaces and because one might have `somethingCaseseWhatever`, I can't just replace `s/ Casese / Cases /`.